docs-gen/content/rule_set/data_entry/data_types.md

```diff
@@ -50,16 +50,12 @@ and the corresponding video stream could look like:
 - Camera.URI:
   type: sensor
   datatype: string
-  description: URI for retrieving the video stream
+  description: URI for retrieving the video stream, with information on how to access the stream (e.g. protocol)
```

Comment:
Sorry to propose a change again but this documentation is a really important part to be able to say we have "solved" the streaming data issue.

What do you think about this for even more clarity:

"...information on how to interpret the stream (e.g. protocol, data format, encoding, etc.)"

Reply:
^^ because I agree with your fundamental point that the main protocol is already in the URI itself.  Only some "tunneled" protocol is valid, and therefore it is good to mentioned here, but there could be also things like video encoding parameters and so on.

Reply:
> Sorry to propose a change again but this documentation is a really important part to be able to say we have "solved" the streaming data issue.

I agree and sometimes things are only noticeable, when you see it in writing. @gunnarx, added the description and rebased. Will squash it during merge into master.